docs/StyleGuide/Style-Guide.md

```diff
@@ -31,47 +38,53 @@ Avoid redundant names when variables or methods are in a class/module.
 ```dafny
 class Integer {
 
-    // The following method converts the given integer
-    // to a string.
-    //
-    // this method name can be simplified to ToString()
-    // so that the method call is Integer.ToString(i)
-    // instead of Integer.IntegerToString(i).
+  // The following method converts the given integer
+  // to a string.
+  //
+  // this method name can be simplified to ToString()
+  // so that the method call is Integer.ToString(i)
+  // instead of Integer.IntegerToString(i).
```

Comment:
Should this comment be moved to the prose above?

Reply:
Done.